app/models/wage_step.rb

```diff
@@ -0,0 +1,3 @@
+class WageStep < ApplicationRecord
+  belongs_to :occupation_standard
```

Comment:
Similar to #37 and #38 can you make the sort order unique within the occupation standard?